client/jetpack-cloud/sections/partner-portal/license-preview/index.tsx

```diff
@@ -56,6 +60,8 @@ export default function LicensePreview( {
 	const licenseState = getLicenseState( attachedAt, revokedAt );
 	const domain = siteUrl ? getUrlParts( siteUrl ).hostname || siteUrl : '';
 
+	const site = useSelector( ( state ) => ( blogId ? getSite( state, blogId ) : null ) );
```

Comment:
Non-blocking:

The `getSite` selector is already written to handle the null case. Then again, I appreciate that this could just be defensive programming. 🙂